docs/tutorials/notes-mvvm/includes/bugs.md

```diff
@@ -0,0 +1,57 @@
+---
+author: adegeo
+ms.author: adegeo
+ms.date: 12/21/2022
+ms.topic: include
+---
+
+Now that the app code can compile and run, you'll likely have noticed that there are two flaws with how the app behaves. The app doesn't let you reselect a note that's already selected, and the list of notes isn't reordered after a note is created or changed.
+
+## Get notes to the top
+
+First, fix the reordering problem with the notes list. In the _ViewModels\\NotesViewModel.cs_ file, the `AllNotes` collection contains all of the notes to be presented to the user. Unfortunately, the downside to using an `ObservableCollection` is that it can't be sorted. You must deal with items in the list one at a time.
```

Comment:
Not entirely sure what you mean by "you must deal with items in the list one at a time"? Reading ahead it looks like you're talking about reordering the list manually?